example/spec/parallel/dbRecords/DeadLetterArchiveProcessingSpec.js

```diff
@@ -131,6 +131,8 @@ describe('A dead letter record archive processing operation', () => {
           },
         }
       );
+      const postRecoverResponseBody = JSON.parse(postRecoverResponse.body);
```

Comment:
If `postRecoverCumulusMessages` returns an error (so no valid `body`), this would blow up, right? But I guess we would see the error in the logs and then another error like `invalid character` or something from `JSON.parse`. Which still gets what you want.

Reply:
Yeah it would still throw an error in that case and be caught as the `beforeAllError` and cause the rest of the tests to fail